### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by guiworldtv2** [iptv-org/community#135] [Q&A] [answered]
What's up
How i can add my channels in this system forked in my channel?
Can i add some channels and the code verify if the channel its off or online and show for me?

**Comment by BellezaEmporium** [comment, marked as answer]
1) Simply follow our [contribution guidelines](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)
2) Use streamtest.in and/or VLC to test if it works, we use the same technology here.
